subsys/net/lib/lwm2m/Kconfig

```diff
@@ -215,6 +215,17 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	  block transfer and "FIRMWARE PACKAGE URI" resource.  This option
 	  adds another UDP context and packet handling.
 
+config LWM2M_SWMGMT_OBJ_SUPPORT
+	bool "Software management object support"
+	default y
+	help
+	  Include support for LWM2M Software Management Object (ID 9)
+
+config LWM2M_SWMGMT_MAX_INSTANCE_COUNT
```

Comment:
condition on SWMGTM_OBJ_SUPPORT, choose a different default prob

Reply:
I thought this was solved by "depends on LWM2M_SWMGMT_OBJ_SUPPORT" below?